core/src/main/resources/error/error-classes.json

```diff
@@ -3685,27 +3685,27 @@
   },
   "_LEGACY_ERROR_TEMP_2043" : {
     "message" : [
-      "- <sqlValue> caused overflow"
+      "- <sqlValue> caused overflow."
     ]
   },
   "_LEGACY_ERROR_TEMP_2044" : {
     "message" : [
-      "<sqlValue1> <symbol> <sqlValue2> caused overflow"
+      "<sqlValue1> <symbol> <sqlValue2> caused overflow."
     ]
   },
   "_LEGACY_ERROR_TEMP_2045" : {
     "message" : [
-      "Unsupported table change: <message>"
+      "Unsupported table change: <message>."
```

Comment:
Please, double check here and other places when a parameter is at the end, it cannot have a dot otherwise we will have double dots.

Reply:
Yeah that's why I keep this PR as draft for double-checking those cases.

Thanks for figuring out!!

Just done double checking and updating the mistaken cases.